legacy_plugins/insight_to_action/package.json

```diff
@@ -0,0 +1,136 @@
+{
+    "name": "insight_to_action",
+    "version": "1.0.2",
+    "description": "Pivot table with button that can call APIs.",
+    "author": "GoodData Corporation",
+    "private": true,
+    "license": "MIT",
```

Comment:
@dremileta point this to the license file as it is done in tooltip: https://github.com/gooddata/gooddata-dashboard-plugins/blob/master/legacy_plugins/tooltip_plugin/package.json#L7, also private true above this is incorrect here.

Reply:
@dremileta since we're in the legacy folder, we need `SEE LICENSE IN ../../LICENSE` instead of `SEE LICENSE IN ../LICENSE`, right?